### DISCUSSION TITLE
Wider local top

### DISCUSSION THREAD

**Discussion by Moltanicaa** [ppy/osu#14084] [Ideas] [answered]
Is it possible to make the panel from the top left go up when scrolling the local top? This will allow you to more comfortably view the local top. Here's an example:
![osu_2021-07-31_16-45-07](https://user-images.githubusercontent.com/59089206/127742004-daa91f25-be98-4e09-9728-5ce6065956ef.png)

**Comment by peppy** [comment, marked as answer]
The screen will eventually be customisable.
